crates/kubelet/src/kubelet.rs

```diff
@@ -99,6 +110,9 @@ impl<P: Provider> Kubelet<P> {
                 },
                 res = plugin_registrar => if let Err(e) = res {
                     error!(error = %e, "Plugin registrar task completed with error");
+                },
+                res = device_manager => if let Err(e) = res {
+                    error!("Device manager task completed with error {:?}", &e);
```

Comment:
Nit: Should use a structured field instead of formatting the error (see line 112)